common/source/docs/common-rfd900.rst

```diff
@@ -9,9 +9,8 @@ designed for long range serial communication.  Ranges of `15km and much higher h
 
 .. warning::
 
-   These modems are high-power RF devices. It is suggested to
-   provide separate +5 voltage to the modems. It is NOT suggested to power
-   these modems directly from Autopilots telemetry port.
+   Many common flight controllers have telemetry port current limits that prevent directly powering RFD900 radios at their full transmit power. You should power high power radios externally or limit their maximum transmit power to an acceptable level. See the manual of your specific flight controller for the detailed power specifications.
```

Comment:
All commonly used flight controllers should be able to handle transmit powers of 20dBm, and most will cope with 25dBm.

Reply:
I think it's best to be more specific and specify the 20dBm or 25dBm (or whatever).  Although most users don't know what a "dBm" is the setup screen has a drop-down for this setting.  Extra points if we include a picture with some red boxes showing how to get to the setting.  I tried to capture a screen shot but my MP is acting up.

Reply:
no need for screen  shot ...easy reference to how to do that already exists....will include in warning...